frontend/src/metabase/env.ts

```diff
@@ -40,6 +45,6 @@ export const isChartsDebugLoggingEnabled = tryOrDefault(
 );
 
 export const isEmbeddingSdk = tryOrDefault(
-  () => !!process.env.IS_EMBEDDING_SDK,
+  () => process.env.IS_EMBEDDING_SDK === "true",
```

Comment:
actually env should be treated as strings, so it's compatible with every environment.